src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -1345,19 +1346,20 @@ export class NotebookModel extends Disposable implements INotebookModel {
 	private sanitizeSavedKernelInfo(): void {
 		if (this._savedKernelInfo) {
 			let displayName = this._savedKernelInfo.display_name;
-
-			if (this._savedKernelInfo.display_name !== displayName) {
```

Comment:
I removed this part since it seems redundant. displayName is already the value of savedKernelInfo.display_name above, so this condition will never get hit.